CHANGELOG.md

```diff
@@ -10,171 +10,218 @@ changes in the following format: PR #1234***
+- Validation for DateElement (JS). (PR #7266)
+- Session Current_stage default value changed for Not Started (PR #7102)
+- Permissions were reorganized and categorized in the user_accounts module (PR #7327)
+- The `EchoTime` field has been added to the following tables: `MRICandidateErrors`, 
+and `mri_violations_log`. `EchoTime` is necessary to distiguish MINC files for 
+multi-echo aquisitions (PR #7515).
+- The `Center_name` field in the `mri_protocol` table has been replaced by `CenterID` 
+from the `psc` table. The default value of `CenterID` is `NULL`. Previously, the 
+default for `Center_name` was `AAAA` or `ZZZZ`. (PR #7525)
 
 #### Bug Fixes
-- The default value of the `ScannerID` field of the `mri_protocol` table is now `NULL` instead of `0`. This means that if a protocol is valid on all the study's scanners, then `ScannerID` of the protocol should be set to `NULL` (PR #7496)
-- The imaging uploader now starts automatically if you have the imaging uploader auto-launch set to true and your current upload overwrites an existing file (PR #7084).
-- Script CouchDB_MRI_Importer now computes the correct names the data dictionary entries associated to MRI comments (PR #7082). 
-- Candidate library now allows a null sex in the select() function to accommodate scanner candidates. This prevents an error from being thrown in the candidate parameters module. (PR #7058)
-- The `EchoTime` field has been added to the following tables: `MRICandidateErrors`, and `mri_violations_log`. `EchoTime` is necessary to distiguish MINC files for multi-echo aquisitions (PR #7515).
-- The `Center_name` field in the `mri_protocol` table has been replaced by `CenterID` from the `psc` table. The default value of `CenterID` is `NULL`. Previously, the default for `Center_name` was `AAAA` or `ZZZZ`. (PR #7525)
-- A LINST instrument Date field name now appears correctly (not truncated) on the instrument if it includes the string `_date`. (PR #6923)
-- A subtest with only static and/or static score fields (i.e. no values to submit) can be saved with no errors so that the instrument scoring script can be called. (PR #7124)
-- When a superuser edits another user, the labels for each permission is correctly displayed (PR #7451) 
+- A LINST instrument Date field name now appears correctly (not truncated) on the 
+instrument if it includes the string `_date`. (PR #6923)
+- A subtest with only static and/or static score fields (i.e. no values to submit) 
+can be saved with no errors so that the instrument scoring script can be called. (PR #7124)
+- When a superuser edits another user, the labels for each permission is correctly 
+displayed (PR #7451) 
+- The imaging uploader now starts automatically if you have the imaging uploader 
+auto-launch set to true and your current upload overwrites an existing file (PR #7084).
+- Script CouchDB_MRI_Importer now computes the correct names the data dictionary 
```

Comment:
```suggestion
- Script `CouchDB_MRI_Importer` now computes the correct names in the data dictionary 
```

Reply:
in or from? not sure. Feels like it is missing something between `names` and `the`